flake.nix

```diff
@@ -112,7 +112,7 @@
           // {
             # The default module adds the default overlay on top of nixpkgs.
             # This is so that `ngipkgs` can be used alongside `nixpkgs` in a configuration.
-            default.nixpkgs.overlays = [self.overlays.default];
+            default.nixpkgs.overlays = [self.overlays.default nix-php-composer-builder.overlays.default];
```

Comment:
Why did you add this overlay here? This output is supposed to only contain modules that are *defined in or deliberately re-exported by* ngipkgs. But you seem to just consume it to use `pkgs.api.buildComposerProject` in `pkgs/flarum/default.nix`. Such a use-case should already be covered by line 51 in `flake.nix`.

Reply:
I'm not entirely sure but we didn't have `buildComposerProject` available to us at the time we were developing this and this allowed us to consume it, so it's possible after rebase on more recent changes that it became redundant. Regardless, we will be removing this to update to use the upstreamed `buildComposerProject`

Reply:
Here's an example on how to use it : https://github.com/NixOS/nixpkgs/pull/255122